pkg/app/app_deploy.go

```diff
@@ -5,7 +5,6 @@ package app
 
 import (
 	"fmt"
-
```

Comment:
nit: This line shouldn't be getting removed. There are some other similar formatting changes that shouldn't be there, could you please check once.